app/models/course/experience_points/forum_disbursement.rb

```diff
@@ -0,0 +1,183 @@
+# frozen_string_literal: true
+class Course::ExperiencePoints::ForumDisbursement < Course::ExperiencePoints::Disbursement
```

Comment:
You want to shorten the class ?  Or you can add a comment to let rubocop ignore it if you think there's nothing can be factored out.